Elemental.JsonResource/JsonResourceGenerator.cs

```diff
@@ -153,71 +157,111 @@ public override bool Execute()
 						{
 							if (!string.IsNullOrEmpty(ns))
 							{
-								w.WriteLine("namespace " + ns + " {");
+								w.WriteLine($"namespace {ns} {{\n");
```

Comment:
What purpose does the '\n' serve? This is a call two WriteLine, so it should have a newline (platform-specific) already. Using '\n' will cause IDEs to complain if someone views the files.
```suggestion
								w.WriteLine($"namespace {ns} {{");
```